drgn/helpers/linux/net.py

```diff
@@ -260,3 +266,88 @@ def skb_shinfo(skb: Object) -> Object:
         return cast("struct skb_shared_info *", skb.head + skb.end)
     else:
         return cast("struct skb_shared_info *", skb.end)
+
+
+def skb_headlen(skb: Object) -> Object:
+    return skb.len - skb.data_len
+
+
+def skb_headroom(skb: Object) -> Object:
+    return skb.data - skb.head
```

Comment:
In the kernel, `skb_headroom()` returns `unsigned int`, but this returns `ptrdiff_t`. If we want to be extra accurate, we should cast this here:

```suggestion
    return cast("unsigned int", skb.data - skb.head)
```